EssentialApp/EssentialAppTests/Helpers/ImageCommentsViewController+TestHelpers.swift

```diff
@@ -0,0 +1,50 @@
+//
+//  ImageCommentsViewController+TestHelpers.swift
+//  EssentialAppTests
+//
+//  Created by Bogdan Poplauschi on 02/04/2021.
+//  Copyright © 2021 Essential Developer. All rights reserved.
+//
+
+import UIKit
+import EssentialFeediOS
+
+extension ImageCommentsViewController {
+	func simulateUserInitiatedImageCommentsReload() {
+		refreshControl?.simulatePullToRefresh()
+	}
+	
+	var isShowingLoadingIndicator: Bool {
+		return refreshControl?.isRefreshing == true
+	}
+	
+	func numberOfRenderedImageComments() -> Int {
+		numberOfRows(in: imageCommentsSection)
+	}
+	
+	func imageCommentView(at row: Int) -> ImageCommentCell? {
+		guard row < numberOfRows(in: imageCommentsSection) else { return nil }
```

Comment:
Use the `numberOfRenderedImageComments` instead:

```suggestion
		guard row < numberOfRenderedImageComments() else { return nil }
```